daemon/logger/journald/read.go

```diff
@@ -365,11 +358,7 @@ func (s *journald) readLogs(logWatcher *logger.LogWatcher, config logger.ReadCon
 	}
 	if config.Follow {
 		cursor = s.followJournal(logWatcher, j, cursor, untilUnixMicro)
```

Comment:
We don't seem to (currently) `close(logWatcher.Msg)` if we get this far - will this new `defer` have a negative effect on `followJournal`?

Reply:
I think that's a mistake in my most recent rebase; the `defer close(logWatcher.Msg)` line at the top of `followJournal` was supposed to be dropped in this commit